sdk/storage/azure-storage-blob/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --java --use:@autorest/java@4.0.x
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-12-06/blob.json
+input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/feature/storage/listBlobsInvalidXmlCharacters/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-02-12/blob.json
```

Comment:
Remember to change this back.  IIRC, STG 80 swagger is now in the main branch - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/storage/data-plane